src/Uno.UI/UI/Xaml/Shapes/Ellipse.cs

```diff
@@ -1,15 +0,0 @@
-﻿#if !__IOS__ && !__MACOS__ && !__SKIA__ && !__ANDROID__
-#define LEGACY_SHAPE_MEASURE
-#endif
-
-using Windows.Foundation;
-
-namespace Windows.UI.Xaml.Shapes
-{
-	public partial class Ellipse
```

Comment:
@dr1rrb The generated partial already unifies the base class. Is this one still needed? It will be almost a duplicate of the generated partial.

Reply:
Ok I'll need it anyway to add the common `MeasureOverride` code there 😄

Reply:
> @dr1rrb The generated partial already unifies the base class. Is this one still needed? It will be almost a duplicate of the generated partial.

Personally I think yes we should keep this file (even if we would not have to add the `MeasureOverride`): the generated files are only "glue code" to match the WinUI API, but eventually when everything will finally been implemented, we should be able to remove them.

And usually I like to see the shared file at the "main entry point for that class" which defines the base type, the class visibility (`public` or `internal`), and hopefully the ctor. Then in the partial files I just add the `partial` modifier on the class and add the specific feature that matches the "sub-file-name".